assets/components/DropdownFilterButton.jsx

```diff
@@ -4,7 +4,12 @@ import {gettext} from 'utils';
 import classNames from 'classnames';
 
 const getActiveFilterLabel = (filter, activeFilter, isActive) => {
-    return isActive ? activeFilter[filter.field][0] : gettext(filter.label);
+    if (isActive) {
+        return filter.transform ? gettext(filter.transform(activeFilter[filter.field][0])) : gettext(activeFilter[filter.field][0]);
```

Comment:
not sure we need to call that gettext together with `filter.transform`, that should be handled there then no?